### PR TITLE
Add latest API Version (October 2020)

### PR DESCRIPTION
**Pull request by gil--**
## Description

Adds the `October20` ApiVersion as an option.

## Type of change

- [X]  Minor: New feature (non-breaking change which adds functionality)

## Checklist

- [ ] I have added a changelog entry, prefixed by the type of change noted above (Documentation fix and Test update does not need a changelog as we do not publish new version)

### CODE REVIEW

**Comment by gil--** [comment]
Do we leave in old supported API versions? Not sure how API Version deprecation works with this library.

**Comment by dahukish** [comment]
> Do we leave in old supported API versions? Not sure how API Version deprecation works with this library.

Going to approve this for now, we can circle back and prune old versions. Could look into a github action that compares latest and triggers a message possibly?

**Review by dahukish — APPROVED** [review]
LGTM

File: packages/koa-shopify-graphql-proxy/CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 <!-- ## [Unreleased] -->
 
+
+## [4.1.0] - 2020-10-14
+
```
Comment (atesgoral) [inline packages/koa-shopify-graphql-proxy/CHANGELOG.md:12]: We should add versions and dates while publishing

```suggestion
```

**Review by atesgoral — APPROVED** [review]

**Pull request merged**